docs/environment-docs.yml

```diff
@@ -3,13 +3,13 @@ channels:
   - conda-forge
 dependencies:
   - python>=3.8
-  - dask[dataframe]
+  - dask[dataframe]<2023.9.2
```

Comment:
```suggestion
  - dask<2023.9.2
```
Not sure how we ever ended up with the square bracket syntax here...

Reply:
Yes I found that rather confusing as well - I didn't touch it in case it was important, but I'll remove it now